docs/core/tools/dotnet.md

```diff
@@ -300,10 +316,27 @@ dotnet myapp.dll
 
   List of assemblies to load and execute startup hooks from.
 
+- `DOTNET_BUNDLE_EXTRACT_BASE_DIR` **Available starting with .NET Core 3.x.**
+
+  Specifies a directory to which a single-file application is extracted before it is executed.
+
+  For more information, see [Single-file executables](../whats-new/dotnet-core-3-0.md#single-file-executables).
+
 - `COREHOST_TRACE`, `COREHOST_TRACEFILE`, `COREHOST_TRACE_VERBOSITY`
 
   Controls diagnostics tracing from the hosting components, such as `dotnet.exe`, `hostfxr`, and `hostpolicy`.
 
+  * `COREHOST_TRACE=[0/1]` - default is `0` - tracing disabled. If set to `1` the diagnostics tracing is enabled.
+  * `COREHOST_TRACEFILE=<file path>` **Available starting with .NET Core 3.x.** - only has effect if tracing is enabled via `COREHOST_TRACE=1`. When set the tracing information is written to the specified file, otherwise the tracing information is written to `stderr`.
```

Comment:
```suggestion
  * `COREHOST_TRACEFILE=<file path>` - only has effect if tracing is enabled via `COREHOST_TRACE=1`. When set, the tracing information is written to the specified file, otherwise the tracing information is written to `stderr`. **Available starting with .NET Core 3.x.** 
```